### ISSUE TITLE
Dataset - datalink not validated against latest schema

### ISSUE THREAD

**Issue by javi-gv94** [inab/benchmarking-data-model#94]
The datalink 'oneOf' structure does not allow to validate datasets which contain uri, attrs...etc
I think it is due to the [additional properties](https://github.com/inab/benchmarking-data-model/blob/1.0.x/json-schemas/1.0.x/dataset.json#L107) found in that object.
Validator output:
```
 Found 4 error(s)

Message:
    Property 'attrs' has not been defined and the schema does not allow additional properties.
Schema path:
    https://www.elixir-europe.org/excelerate/WP2/json-schemas/1.0/Dataset#/properties/datalink/additionalProperties

Message:
    Property 'status' has not been defined and the schema does not allow additional properties.
Schema path:
    https://www.elixir-europe.org/excelerate/WP2/json-schemas/1.0/Dataset#/properties/datalink/additionalProperties

Message:
    Property 'uri' has not been defined and the schema does not allow additional properties.
Schema path:
    https://www.elixir-europe.org/excelerate/WP2/json-schemas/1.0/Dataset#/properties/datalink/additionalProperties

Message:
    Property 'validation_date' has not been defined and the schema does not allow additional properties.
Schema path:
    https://www.elixir-europe.org/excelerate/WP2/json-schemas/1.0/Dataset#/properties/datalink/additionalProperties
```

**Comment by javi-gv94** [comment]
Here is an example of a file which is not validated (sorry json format is not allowed..)
[Dataset_participant_ACC_2020plus_OEBD002t000003.txt](https://github.com/inab/benchmarking-data-model/files/2931068/Dataset_participant_ACC_2020plus_OEBD002t000003.txt)

**Comment by jmfernandez** [comment]
This issue has just been fixed at 1aabbef3190acb46341a2af87c114f773e357575 . Due the semantics of `additionalProperties`, in `oneOf` cases it must be used on each one of the alternatives, instead of the parent one
